Automattic/jetpack
# Add Jetpack AI Yearly to Jetpack plugin

Related to #30543

## Proposed changes:
- Replace slug used for purchase (e.g. in interstitial) from _monthly to _yearly
- Add JETPACK_AI_YEARLY to constants and icons

## Testing instructions:

- Checkout the PR and run Jetpack locally (docker, `jetpack watch plugins/jetpack`)
- Go to `/wp-admin/admin.php?page=my-jetpack#/add-jetpack-ai`
- Make sure that the annual price is displayed:

<img width="1125" alt="CleanShot 2023-08-16 at 15 51 06@2x" src="https://github.com/Automattic/jetpack/assets/8419292/039b78b6-d452-468e-852e-8065d307fcb8">
